src/datacontroller.js

```diff
@@ -204,4 +212,31 @@ export default class DataController {
 	 * Removes all event listeners set by the DataController.
 	 */
 	destroy() {}
+
+	/**
+	 * See {@link engine.model.composer.insertContent}.
+	 *
+	 * @fires engine.DataController#insertContent
+	 * @param {engine.model.Batch} batch Batch to which deltas will be added.
+	 * @param {engine.model.Selection} selection Selection into which the content should be inserted.
+	 * @param {engine.view.DocumentFragment} content The content to insert.
+	 */
+	insertContent( batch, selection, content ) {
```

Comment:
I agree with I think everything here, but I want to work on a cleanup when we'll be moving `deleteContent()` here. In fact – should it be `delete()` then too? I think this would be confusing so the `Content` part is good in both cases.